src/test/java/io/confluent/idesidecar/restapi/application/SidecarInfoTest.java

```diff
@@ -58,186 +65,163 @@ public static SidecarInfo createSidecarInfo(TestInputs input) {
-                vscodeVersion,
-                SidecarInfo.VERSION
-            )
+            "Confluent-for-VSCode/v1.2.3 (https://confluent.io; support@confluent.io) sidecar/v5.40.3 (windows/10.1.X.Y)"
         ),
         new TestInputs(
             "Windows 11 info with VS Code and version prefix",
-            "Windows 11",
-            "10.0.1904562",
+            "windows",
+            "10.1.X.Y",
+            "20.1.2",
             OperatingSystemType.Windows,
             new SidecarInfo.VsCode("20.1.2", "1.2.3"),
             "v",
-            "Confluent-for-VSCode/v%s (https://confluent.io; support@confluent.io) sidecar/v%s (Windows 11/x86_64)".formatted(
-                vscodeVersion,
-                SidecarInfo.VERSION
-            )
+            "Confluent-for-VSCode/v1.2.3 (https://confluent.io; support@confluent.io) sidecar/v5.40.3 (windows/10.1.X.Y)"
         ),
         new TestInputs(
             "Windows 11 info without VS Code",
-            "Windows 11",
+            "windows",
             "10.1.X.Y",
+            "20.1.2",
             OperatingSystemType.Windows,
             null,
             null,
-            "Confluent-for-VSCode/v%s (https://confluent.io; support@confluent.io) sidecar/v%s (Windows 11/x86_64)".formatted(
-                vscodeVersion,
-                SidecarInfo.VERSION
-            )
+            "Confluent-for-VSCode/v1.2.3 (https://confluent.io; support@confluent.io) sidecar/v5.40.3 (windows/x86_64)"
         ),
         // Other
         new TestInputs(
             "Solaris info with VS Code",
-            "Solaris 4",
+            "solaris",
             "4.1.X.Y",
+            "20.1.2",
             OperatingSystemType.Other,
             new SidecarInfo.VsCode("20.1.2", "1.2.3"),
             "",
-            "Confluent-for-VSCode/v%s (https://confluent.io; support@confluent.io) sidecar/v%s (Solaris 4/x86_64)".formatted(
-                vscodeVersion,
-                SidecarInfo.VERSION
-            )
+            "Confluent-for-VSCode/v1.2.3 (https://confluent.io; support@confluent.io) sidecar/v5.40.3 (solaris/4.1.X.Y)"
         ),
         new TestInputs(
             "Solaris info with VS Code and version prefix",
-            "Solaris 4",
+            "solaris",
             "4.1.X.Y",
+            "20.1.2",
             OperatingSystemType.Other,
             new SidecarInfo.VsCode("20.1.2", "1.2.3"),
             "v",
-            "Confluent-for-VSCode/v%s (https://confluent.io; support@confluent.io) sidecar/v%s (Solaris 4/x86_64)".formatted(
-                vscodeVersion,
-                SidecarInfo.VERSION)
+            "Confluent-for-VSCode/v1.2.3 (https://confluent.io; support@confluent.io) sidecar/v5.40.3 (solaris/4.1.X.Y)"
         ),
         new TestInputs(
             "Solaris info without VS Code",
-            "Solaris 4",
+            "solaris",
```

Comment:
We don't have an enum literal for `SOLARIS`, or more correctly `UNIX`, in `OperatingSystemType`. If we want to distinguish, we should add the enum literal, and update the test.